src/Moryx.Products.Management/Components/IProductManager.cs

```diff
@@ -100,7 +100,7 @@ internal interface IProductManager : IAsyncPlugin
         /// </summary>
         /// <param name="ids">The IDs of instances that should be loaded</param>
         /// <returns>The instance with the id when it exists.</returns>
-        Task<IReadOnlyList<ProductInstance>> GetInstances(params long[] ids);
+        Task<IReadOnlyList<ProductInstance>> GetInstances(long[] ids);
```

Comment:
Why did you change this? 🤔

Reply:
Because the public api was also with params and passing cancellationtoken with default in combination was irritating in the api usage. The api user can pass an array too